scripts/arrays.js

```diff
@@ -0,0 +1,12 @@
+let  arrayOfNames = ["green", "tree", "22", "bag", "10.10", "house"];
+
+alert(arrayOfNames[0]);
+alert (arrayOfNames[arrayOfNames.length-1]);
+
+color = ["Blue ", "Green", "Red", "Orange", "Violet", "Indigo", "Yellow "];
+o = ["th","st","nd","rd"];
+
+alert(`1${o[1]} choice is ${color[0]}.
+2${o[2]} choice is ${color[1]}.
```

Comment:
ესეც იმუშავებს, მაგრამ უკეთესი იქნებოდა \n newLine გამოგეყენებინა
https://www.simplilearn.com/tutorials/javascript-tutorial/javascript-new-line
ამას გადახედე
console.log(`1${o[1]} choice is ${color[0]}. \n2${o[2]} choice is ${color[1]}. \n3${o[3]} choice is ${color[2]}`)
ეს კოდიც ნახე და გაარჩიე როგორ მუშაობს \n